aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -395,7 +389,7 @@ def get_writer(file_format: str) -> Type[ArchiveWriterAbstract]:
             f'Can only export in the formats: {tuple(writers.keys())}, please specify one for "file_format".'
         )
 
-    return writers[file_format]
+    return cast(Type[ArchiveWriterAbstract], writers[file_format])
```

Comment:
this is required due to the differing __init__ signatures of the different implementations (with specific keywords like `use_compression`)